Runtime/src/index.ts

```diff
@@ -226,6 +226,44 @@ export class SwiftRuntime {
                 this.memory
             );
         },
+        swjs_call_function_unsafe: (
```

Comment:
Also, maybe call these `swjs_call_function_no_catch` to more clearly say what they do? Since they don’t exhibit undefined behavior or cause memory safety issues, I think they need to be called “unsafe.” But I’m definitely not 100% confident here and would like to hear your thoughts!

Reply:
I like your idea! I've added a new commit renaming `unsafe` to `no_catch` 😃